scoring/score.py

```diff
@@ -1,9 +1,57 @@
+DATA = {
+    # Location -> (Owner, Points if owner matches)
+    'arena':              (None, 0),
+    'docking_area':       (None, 1),
+    'zone_0_raised_area': (0,    3),
+    'zone_1_raised_area': (1,    3),
+}
```

Comment:
I think it might be a good idea to add type annotations onto this? A newtype/alias or two would specify in code what is currently in the comment.

Reply:
There's no type checking in this repo, so the benefits are limited. Will add a comment though.